tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -259,7 +260,7 @@ def test_handle_merge_request_no_deprecate(
 @mock.patch('iib.workers.tasks.build_merge_index_image._opm_index_add')
 @mock.patch('iib.workers.tasks.build_merge_index_image.set_request_state')
 def test_add_bundles_missing_in_source(
-    mock_srs, mock_oia, mock_aolti, mock_bi, mock_pi, mock_capml, mock_gil
+    mock_srs, mock_oia, mock_aolti, mock_bi, mock_pi, mock_capml, mock_gil, mock_iifbc
```

Comment:
May I suggest adding a test when is_image_fbc is true

Reply:
I will add it in deprecate PR - I am not able to do it here because it is raising IIBError because deprecation does not support FBC yet.

Reply:
It would have been better to have a test without the deprecation involved, but its not a blocker for me